scripts/compress_assets.sh

```diff
@@ -7,9 +7,9 @@ set -euo pipefail
 cd ui/react-app
 cp embed.go.tmpl embed.go
 
-GZIP_OPTS="-fk"
+GZIP_OPTS="-fkn"
```

Comment:
`-n` means to avoid including file naming information.

This also happens to exclude file timestamp info, which keeps the built artifacts reproducible. Otherwise, you get diff in the compressed files every time they're rebuilt.